raiden/ui/cli.py

```diff
@@ -909,8 +909,10 @@ def run(self):
         )
         if not self._options['accept_disclaimer']:
             click.confirm('\nHave you read and acknowledge the above disclaimer?', abort=True)
+
+        log_config = configure_logging_defaults(self._options['log_config'])
```

Comment:
I disagree, because cli.py is a large module doesn't mean that code should not be split into functions with different responsibilities. What is wrong the split?